### ISSUE TITLE
Result.activation_frame_hz is unpopulated despite activations being emitted

### ISSUE THREAD

**Issue by eversonl** [openmirlab/all-in-one-infer#2]
### Summary

`allin1fix.analyze()` returns a `Result` whose `activations` attribute is populated but whose `activation_frame_hz` attribute is `None`. Downstream consumers can't map activation indices back to timestamps without the frame rate.

### Reproduction

```python
import torch
# CPU-only host workaround (separate concern):
if not torch.cuda.is_available():
    torch.cuda.get_device_capability = lambda device=None: (0, 0)

import allin1fix
result = allin1fix.analyze("any-track.wav")

print(result.activations is not None)                # True
print(getattr(result, "activation_frame_hz", None))  # None  <-- bug
print(getattr(result, "activation_fps", None))       # None
```

### Expected

`result.activation_frame_hz` should be a float matching the model's actual frame rate. For `harmonix-all` this is 100 Hz (44100 sample rate / 441 hop):

```python
from allin1fix.models import load_pretrained_model
m = load_pretrained_model("harmonix-all", device="cpu")
print(m.cfg.fps)         # 100
print(m.cfg.sample_rate) # 44100
print(m.cfg.hop_size)    # 441
```

### Impact

Without the field, every downstream consumer must either hardcode a constant (silently breaks if the model variant changes) or derive it from `len(activations.beat) / audio_duration` (small drift due to model padding).

We've observed real-world clients that hardcoded `40.0` (presumably copied from some example output that was wrong from the start). Those clients have been **2.5× off** when computing `time = activation_index / activation_frame_hz` — small enough to look plausible, large enough to cause cumulative misalignment on long tracks.

### Suggested fix

In whichever module constructs the `Result`, populate `activation_frame_hz` from the loaded model's `cfg.fps`:

```python
result.activation_frame_hz = float(model.cfg.fps)
```

Happy to open a PR if pointed at the right call site.

### Workaround in our consumer

We've added a server-side fallback that lazily loads the model itself to read `cfg.fps`, and fills in the field whenever upstream returns null. See [beatserver fix/app.py](https://github.com/eversonl/Beatserver/blob/main/fix/app.py) and [legacy/app.py](https://github.com/eversonl/Beatserver/blob/main/legacy/app.py) for the pattern. The fallback is fail-safe (load failure keeps the field null), forward-compatible (honours any value upstream populates), and only adds one model load per process lifetime. We'd prefer to retire that workaround once upstream populates the field at the source.

Filed alongside the existing NATTEN-perf issue #1.

Thank you for maintaining the fork — the `stem_provider` extension is a real productivity unlock for our pipeline.

**Comment by ChenPaulYu** [comment]
Fixed in **v3.0.1** (on PyPI now): `AnalysisResult` gains `activation_fps` — the frame rate (Hz) of the `activations` array, populated whenever activations are requested. Your repro's fallback check was prescient: we went with `activation_fps` (it matches the internal `cfg.fps` naming, and the value comes from the model's own config rather than a hardcoded 100, so it stays correct for any checkpoint). Saved-JSON round-trip is backward compatible — older results load with `None`.

```python
result = allin1_infer.analyze("track.wav", include_activations=True)
t = frame_index / result.activation_fps
```

Note the 3.0.0 rename: the package/import is now `all-in-one-infer` / `allin1_infer` (formerly all-in-one-fix / allin1fix). Also: your `torch.cuda.get_device_capability` monkeypatch is no longer needed on 3.x — that call came from NATTEN's internals, and NATTEN is no longer a dependency (pure-PyTorch neighborhood attention by default). Thanks for the sharp report!
